health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/AddOrEditMapping.js

```
@@ -62,7 +62,10 @@ const AddOrEditMapping = forwardRef(({ schema, dispatch,  boundaryHierarchy, all
             {
                 return (
                     <div style={{ marginBottom: "1rem" }}>
-                        <label>{t("CHOOSE_BOUNDARY_LEVEL")}</label>
+                        <div style={{display:"flex"}}>
+                            <label>{t("CHOOSE_BOUNDARY_LEVEL")}</label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)